python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -220,10 +232,25 @@ def _DataFrame__dir__(self):
         "_constructor": _FastSlowAttribute("_constructor"),
         "_constructor_sliced": _FastSlowAttribute("_constructor_sliced"),
         "_accessors": set(),
+        "_ipython_canary_method_should_not_exist_": custom_func,
     },
 )
 
 
+def custom_repr_html(obj):
+    # This custom method is need to register a html format
+    # for ipython
+    return _fast_slow_function_call(
+        lambda obj: obj._repr_html_(),
+        obj,
+    )[0]
+
+
+if ipython_shell:
+    html_formatter = ipython_shell.display_formatter.formatters["text/html"]
```

Comment:
Guessing this is from https://ipython.readthedocs.io/en/stable/config/integrating.html#formatters-for-third-party-types?

Could you add a comment here with this link?

Reply:
Done